paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetConfigurationValidator.kt

```diff
@@ -0,0 +1,21 @@
+package com.stripe.android.paymentsheet
```

Comment:
nit: when defining an extension, it's usually named as `ClassNameKtx.kt` such as [StripeIntentKtx](https://github.com/stripe/stripe-android/blob/c294238538f0a6a1601228c263a78bf7b110adb8/payments-core/src/main/java/com/stripe/android/model/StripeIntentKtx.kt#L5) and [StripeKtx](https://github.com/stripe/stripe-android/blob/ad426ca33105e7bc5f17c027f8146354d13f75ae/payments-core/src/main/java/com/stripe/android/StripeKtx.kt#L32-L31)